js/search.js

```diff
@@ -155,7 +155,7 @@
 		var queryAlias;
 		if (query in BattleAliases) {
 			queryAlias = toId(BattleAliases[query]);
-			if (query === 'pika' || query === 'bee' || query === 'sab') searchPasses.shift();
+			if (queryAlias.search(query) === 0) searchPasses.shift();
```

Comment:
I have tested it and it worked for every alias as far as I could see. Also I guess .indexOf would be better here, gonna change that.